src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -289,6 +290,11 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
             {
                 throw new Exception("Executing given and whens took longer than 2 minutes");
             }
+
+            if (completedTask.IsFaulted && completedTask.Exception != null)
+            {
+                ExceptionDispatchInfo.Capture(completedTask.Exception).Throw();
```

Comment:
on second thought, does `Capture(ex).Throw()` make sense? wouldn't that be the same as `throw completedTask.Exception`?